project/Version.scala

```diff
@@ -37,6 +40,6 @@ object Version {
   val scalacheckShapeless    = "1.2.5"
   val diffx                  = "0.4.0"
   val log4j                  = "2.15.0"
-  val kubernetesApi          = "15.0.1"
+  val kubernetesApi          = "21.0.0-legacy"
```

Comment:
Upgrading this past version 17 will fix the snake.yaml cve issue. I'm using `21.0.0-legacy` version here because we would need to make significant code changes if we used the 21.0.0 version, since that version makes a method we rely on private instead of public.